bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js

```diff
@@ -3,6 +3,33 @@ import { check, Match } from 'meteor/check';
 import Meetings from '/imports/api/meetings';
 import Logger from '/imports/startup/server/logger';
 
+const getLockSettings = props => {
+  if (props) {
+    // Since we miss a name convention here, we need to translate some properties
+    // key name to fit the rest of the messaging system
+    props = Object.assign(props, {
+      disablePrivChat: props.disablePrivateChat,
```

Comment:
If I understand this correctly, we could adapt the html5 name to match the convention

Reply:
Yes, we could. But we have to take a special care about things like this:
https://github.com/bigbluebutton/bigbluebutton/blob/3775d1ebc2c3334af350562f9c9288dd79ad9b13/bigbluebutton-html5/imports/api/meetings/server/methods/toggleLockSettings.js#L25-L34